apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -447,15 +449,31 @@ export const setCoteacherInvite = coteacherInvite => ({
   coteacherInvite,
 });
 
+export const setCoteacherInviteForPl = coteacherInviteForPl => ({
+  type: SET_COTEACHER_INVITE_FOR_PL,
+  coteacherInviteForPl,
+});
+
 export const asyncLoadCoteacherInvite = () => dispatch => {
   fetchJSON('/api/v1/section_instructors')
     .then(sectionInstructors => {
-      // Find the oldest invite.
-      const coteacherInvite = sectionInstructors.find(
-        instructor => instructor.status === 'invited'
+      const coteacherInviteForPl = sectionInstructors.find(instructorInvite => {
+        return (
+          instructorInvite.status === 'invited' &&
+          instructorInvite.participant_type === 'teacher'
+        );
+      });
+      const coteacherInviteForClassrooms = sectionInstructors.find(
+        instructorInvite => {
+          return (
+            instructorInvite.status === 'invited' &&
+            instructorInvite.participant_type === 'student'
```

Comment:
As a safety precaution, maybe we should use `!== 'teacher'` instead of `=== 'student'` just in case we add a new participant type in the future or there is some parsing error and we still want to see the invites.

Reply:
We do have a "facilitator" participant type currently... any courses who have a "facilitator" as a participant goes into the "Professional Learning" section.  For that reason, we will want to keep the `=== student` for sure.  I think it could make sense to do `!== 'student'` for `facilitator/teacher` participants.  @dju90  can you confirm that sounds good to you?

Reply:
Just to clarify, would that be changing line 463 to `!== 'student'`?

Reply:
If so, that sounds good to me. It ensures that sections with either teachers as participants or facilitators as participants appear in the PL sections area, correct?

Reply:
Correct and correct!  I will make that change!